packages/core/src/servient.ts

```diff
@@ -30,6 +30,9 @@ export default class Servient {
     private things: Map<string, ExposedThing> = new Map<string, ExposedThing>();
     private credentialStore: Map<string, Array<unknown>> = new Map<string, Array<unknown>>();
 
+    private started = false;
+    private shutdowned = false;
```

Comment:
Hmm, maybe we could also use `closed` as a variable name here?

Reply:
I don't like "shutdowned" either and had "destroyed" at first but what convinced me was to align it with the function names we use...

Reply:
Hmm, but I guess the correct adjective form for "to shut down" would be "shut down", right? How about something like `isShutDown`? Since its private, we could probably also use `#shutdown` or `#shutDown` for this field without causing a conflict.

Reply:
like this https://github.com/eclipse-thingweb/node-wot/pull/1195/commits/08e85536a462be9c2fed6abd01304ac9f0827dc7